jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/resourceErrorHandling.feature

```diff
@@ -1,23 +1,23 @@
 Feature: Testing errors caused by invalid template, meta data or file type
 
-  Scenario: Deploy a war file compressed using 7z
```

Comment:
Why remove ? I think this scenario was written to test a bug that we had with regards to incorrect zip.

Reply:
Hi I have included this in another scenario for testing of error in webapp resource, so that we don't duplicate creating jvm and webapp

Reply:
Is it compressed using 7z also ?

Reply:
yeah lines 28-33 and 43-46